sdk/servicebus/service-bus/src/queueClient.ts

```diff
@@ -155,12 +156,16 @@ export class QueueClient implements Client {
 
   /**
    * Creates a Receiver for receiving messages from a Queue which does not have sessions enabled.
-   * Throws error if an open receiver already exists for this QueueClient.
-   *
-   * Throws error if the Queue has sessions enabled.
+   * - Throws error if an open receiver already exists for this QueueClient.
+   * - Throws error if the Queue has sessions enabled.
    *
    * @param receiveMode An enum indicating the mode in which messages should be received. Possible
-   * values are `ReceiveMode.peekLock` and `ReceiveMode.receiveAndDelete`
+   * values are:
+   * - `ReceiveMode.peekLock`: Once a message is received in this mode, the reciever has a lock on
```

Comment:
`reciever` -> `receiver`

Reply:
maybe we should add a linter rule just for this, because it looks like I am simply incapable of spelling this word right .... :(